third_party/terraform/data_sources/data_source_tpu_tensorflow_versions.go

```diff
@@ -0,0 +1,81 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"sort"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func dataSourceTpuTensorflowVersions() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTpuTensorFlowVersionsRead,
+		Schema: map[string]*schema.Schema{
+			"project": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"zone": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
```suggestion
				Optional: true,
				Computed: true,
```

I don't think this does anything, but I'd slightly prefer it for consistency with `project`.